File: 101 Dalmatians - squash the bugs, not the dogs.py
def how_many_dalmatians(number):
  dogs = ["Hardly any", "More than a handful!", "Woah that's a lot of dogs!", "101 DALMATIONS!!!"]
  respond = 0
  if number <= 10:
    respond = dogs[0]
  elif (number <= 50):
    respond = dogs[1]
  elif (number <= 100):
    respond = dogs[2]
  else:
    respond = dogs[3]  
  return respond

File: 101 Dalmatians - squash the bugs, not the dogs.ts
export function howManyDalmatians(num : number): string{
  
    let dogs = ["Hardly any", "More than a handful!", "Woah that's a lot of dogs!", "101 DALMATIANS!!!"];

    let respond = num <= 10 ? dogs[0] : (num <= 50 ? dogs[1] : (num == 101 ? dogs[3] : dogs[2]))

    return respond
}

